src/releases/views/StatsView.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Helmet} from 'react-helmet';
+import {useQuery} from '@apollo/client';
+import {BASIC_RELEASES} from '../queries';
+import {Container, Header, Loader, Segment} from 'semantic-ui-react';
+import {ReleaseBarChart} from '../components/ReleaseBarChart';
+import {ReleaseActivity} from '../components/ReleaseActivity';
+
+const StatsView = ({match}) => {
+  const {
+    data: releasesData,
+    loading: releasesLoading,
+  } = useQuery(BASIC_RELEASES, {fetchPolicy: 'network-only'});
+
+  const releases =
+    releasesData && releasesData.allReleases.edges.map(({node}) => node);
+
+  return (
+    <Container as={Segment} basic vertical>
+      <Helmet>
+        <title>{`KF Data Tracker - Releases`}</title>
+      </Helmet>
+
+      <Header as="h1" className="noMargin">
```

Comment:
How about something like this to give more visual separation between the headers:

![image](https://user-images.githubusercontent.com/10283712/107279377-f82d0900-6a24-11eb-9768-d897cf9a4885.png)

I removed the `noMargin` style on the first header and used `color=grey` style on both headers

Reply:
I agree that the margin shouldn't be removed here. Greying the text is a good practice for de-emphasizing without modifying text size, but I'm hesitant to do this because we don't follow this pattern with headers on other pages.